### ISSUE TITLE
Hyperparamter(self.LA, self.EC, self.EA) not used

### ISSUE THREAD

**Issue by sh-qiangchen** [divelab/LECI#5]
Hi, Thanks for sharing your code.
The hyperparamter(self.LA, self.EC, self.EA) is not used in the LECI model, please check this model's code again!
`
self.LA = config.ood.extra_param[0]
self.EC = config.ood.extra_param[1]
self.EA = config.ood.extra_param[2]
`

**Comment by CM-BF** [comment]
Hi Qiang,

Thank you for your issue. Please check the following code for the usage of these hyper-parameters.

- [In OOD algorithm management](https://github.com/divelab/LECI/blob/11a9c03296d2bb4c908c676eb6bd5687d8c5d40e/GOOD/ood_algorithms/algorithms/LECI.py#L114)
- [In model](https://github.com/divelab/LECI/blob/11a9c03296d2bb4c908c676eb6bd5687d8c5d40e/GOOD/networks/models/LECIGNN.py#L122)

Please let me know if any further questions. :)

Best,
Shurui

**Comment by sh-qiangchen** [comment]
Thanks for your reply. 
For GOOD-CMNIST datasets, I find after a very few epoches of LECI training, the EA and EF loss are close to 0. Doesn't these loss play no role in the subsequent training process? 
And I can't reproduce the results for GOOD-CMNIST datasets, can you provide some hyperparameter settings that can be reproduced?
I used hyperparameter settings：
![image](https://github.com/user-attachments/assets/920a60cc-4cc9-4c72-af25-e70550cc807d)

**Comment by CM-BF** [comment]
Thank you for the question and details! To clarify, the reproducible config is located in [final_configs](https://github.com/divelab/LECI/tree/LECI-1.0.0/configs/final_configs), while the `GOOD_configs` only acts as the default config template with hyperparameter explanations.
